Taskfile.yml

```
@@ -132,6 +132,30 @@ tasks:
     cmds:
       - task dev:cli -- drush cache:rebuild-external -y
 
+  dev:dpl-react:get-asset-url:
+    summary: Outputs the url to the build in Github
+    cmds:
+      - ./dev-scripts/get-dpl-react-assets.py {{ .BRANCH }}
+    preconditions:
+    - sh: "[ ! -z {{.BRANCH}} ]"
+      msg: "Env variable BRANCH is not set or empty."
+
+  dev:dpl-react:download:
```

```suggestion
  dev:dpl-react:overwrite:
```

This stresses the purpose of the task is to overwrite an existing version of the PR - not just download it.